modules/core-services/endpoints/organization.go

```diff
@@ -477,9 +477,9 @@ func (e *Endpoints) convertToOrgDTO(org model.Org, domains ...string) apistructs
 	if len(domainAndPort) > 1 {
 		port = domainAndPort[1]
 	}
-	concatDomain := strutil.Concat(strutil.ToLower(org.Name), "-org.", conf.RootDomain())
+	concatDomain := conf.RootDomain()
```

Comment:
I think UI_DOMAIN is correct, not ROOT_DOMAIN

Reply:
It makes sense.